### ISSUE TITLE
Query execution clears multiFilter

### ISSUE THREAD

**Issue by lockyStar** [getredash/redash#2351]
### Issue Summary

Query execution clears multiFilter choices. That leads to some difficulties when you got more than 10 options to choose all the time executing same query with different parameters.

Solution ideas:
- All the multiFilter options can be replaced by new items after updating query, but it is possible to use intersection of both sets of options. You just need to store last one used.
- Add button that stores current multiFilter state and after each refreshing multiFilter tries to pick up all of the stored options.

### Technical details:

* Redash Version: 4.0.0-beta

**Comment by edowdy** [comment]
Yes this is still an issue in 5.02! Big hit to UX.  Thanks for hopefully getting this fixed ASAP.
